data/hai missions.txt

```diff
@@ -1053,3 +1053,250 @@ mission "Hai Honeymoon"
+			`	They tell Nanachi that they won't be going along, they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`
+				accept
+	on stopover "Alta Hai"
+		dialog
+			`	A spaceport worker contacted Nanachi shortly before you two landed, and made sure the spices were ready to be loaded in her ship. You help the workers and Nanachi make sure everything is in order, and prepare for you trip to Allhome.`
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		conversation
+			`Allhome's docking area shows much more traffic than usual, many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You and Nanachi meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`
+
+
+
+mission "Nanachi 4"
+	name "Protect Nanachi"
+	description "Bring Nanachi's ship to Prime, keeping her safe from a pirate fleet that attacked a convoy heading to Hai space, then head back to <destination>."
+	source "Allhome"
+	stopover "Prime"
+	deadline 14
+	to offer
+		has "Nanachi 3: done"
+	on offer
+		conversation
+			`The bar is just as busy as the docks, many captains here must have transported something for the festival. "Captain <last>, captain Nanachi!" Narancia waves at you two and becons you to their table. You learn more about the festival, as they tell you that they were inspired by the one the Hai have during March, and wanted to bring small parts of human culture to Hai space.`
+			`	"They can't go to our planets and experience it for themselves yet, so we thought about bringing it to them!" Vanessa says.`
+			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the european renaissance!" Xavier adds.`
+			`	Afterwards the conversation topic shifts to you as a captain, as you tell them about your journey after buying your first ship, and how you met Nanachi. You're about to order another drink for the group when a grizzled old man barges into the bar, scouting the area until he spots your group, and starts shouting: "The art convoy has been attacked! The freighter managed to escape on time but they're stranded on Prime now, the pirates are patrolling the route to the wormhole!"`
```

Comment:
`tell them about` -> `describe`

`scouting the area until he spots your group` seems unnecessary

`escape on time` -> `escape`

`Prime now, the pirates are patrolling the route to the wormhole!` -> `Prime now, and the pirates are still out there, patrolling the route to the wormhole!`